doc/concepts/hybrid_planning/hybrid_planning.rst

```diff
@@ -0,0 +1,171 @@
+===============
+Hybrid Planning
+===============
+
+MoveIt’s motion planning architecture follows the “Sense-Plan-Act” approach. In order for motions to get planned and executed, the environment and the robot state are first perceived (“Sense”), then the robot trajectory is computed by the planner (“Plan”) and finally executed in a single run using a trajectory controller (“Act”).
```

Comment:
```suggestion
MoveIt’s motion planning architecture previously followed the “Sense-Plan-Act” approach. In order for motions to get planned and executed, the environment and the robot state are first perceived (“Sense”), then the robot trajectory is computed by the planner (“Plan”) and finally executed in a single run using a trajectory controller (“Act”).
```

Reply:
I think it still does, adding "previously" might be misleading as people could think that this automagically works with the planning pipeline.